kibana/README.md

```diff
@@ -996,3 +996,64 @@ readonlyrest:
 ```
 
 Now try to login as user1, and ReadonlyREST Enterprise should initialise the index ".kibana\_user1" with all the index patterns and dashboards contained in the template tenancy.
+
```

Comment:
@mateuszkp96 to describe the configuration of ROR settings related to the Impersonation feature you can use [this](https://github.com/beshu-tech/readonlyrest-docs/blob/de65586040c8c20c2f77fa725bdf567d60d6f969/elasticsearch-details/impersonation.md). We don't have to mention any more about the requests format (now we treat it as our internal API)